src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($day-size + $gap-size) * 7 - $gap-size + $padding-size * 2;
+  padding: $padding-size;
+  gap: $gap-size;
+
+  &__day {
+    background-color: $c-grey;
+    border: 1px solid $c-black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $day-size;
+    height: $day-size;
+    font-size: 30px;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: $c-pink;
+      cursor: pointer;
+      transform: translateY(-20%);
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: $i + "";
+      }
+    }
+  }
+
+  @for $i from 28 through 30 {
+    &--month-length_#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $i in $days {
+    &--start-day_#{$day} :first-child {
+      margin-left: $i * ($day-size + $gap-size);
+    }
+  }
```

Comment:
It doesn't work right, because you don't add box-sizing: border-box on your calendar__day

Reply:
thanks))